modules/objdetect/test/test_qrcode_encode.cpp

```diff
@@ -33,6 +33,18 @@ std::string encode_qrcode_eci_images_name[] = {
 const Size fixed_size = Size(200, 200);
 const float border_width = 2.0;
 
+void randomShuffle(std::string& info) {
+#if defined CV_CXX11
```

Comment:
I thought we are going to remove `CV_CXX11` completely, see #23736
**cc** @asmorkalov , @opencv-alalek , @vpisarev

Reply:
Indeed. Plus, OpenCV 4.x is C++11 so we can just use std::shuffle by default.